### DISCUSSION TITLE
How to Import any NFT into Zeromint

### DISCUSSION THREAD

**Discussion by treeder** [zeromintapp/community#17] [Show and tell]
Just use the following URL format in your browser:

```
https://zeromint.com/nfts/CHAIN/CONTRACT_ADDRESS/TOKEN_ID
```

See some examples in this Twitter thread: https://twitter.com/zeromintapp/status/1501305336745058307

**Comment by wwitkdqsck** [comment]
CHAIN=Ticker Of The CHAIN?

GoChain=GO
Binance Smart Chain=BSC
Kucoin Community Chain=KCS 
Polygon Chain =MATIC
as long its EVM compatible etc...?

**Comment by wwitkdqsck** [comment]
And for this New Url ...There is no Collection Owner?And No Banner as well ?Because we can't make Changes ...
For Polygon Both erc 1155 and erc 721 works?

**Reply by gobullsclub** [reply]
And How do i make the New Way (Url Way)  ( Collection) Private ? In case we mint it wrongly...?

**Reply by treeder** [reply]
You can't technically, blockchain is blockchain, but it shouldn't show up anywhere unless you share it.

**Reply by wwitkdqsck** [reply]
👍 Noted!

**Comment by gobullsclub** [comment]
Can I import  erc1155?

**Reply by gobullsclub** [reply]
Hello @treeder, @r-gochain, And in the future 🔮, can a collection owner set a banner to the collection?

**Reply by gobullsclub** [reply]
And is Erc721A supported as well?
